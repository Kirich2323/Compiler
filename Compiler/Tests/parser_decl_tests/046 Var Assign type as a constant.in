const
    b = 10;
var
    a : b;