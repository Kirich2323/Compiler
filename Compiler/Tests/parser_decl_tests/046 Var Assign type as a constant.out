(4 ; 10): Got unexpected token: ";". Expected: "..".